integrationtests.py

```diff
@@ -707,8 +698,7 @@ def testObsoleteHeaderDisappears(self):
             self.assertEqual(misses3, misses2)
             self.assertEqual(hits3, hits2+1)
 
-
-class TestRunParallel(unittest.TestCase):
+class TestRunParallelBase:
```

Comment:
Uh-oh... defining base classes purely for the sake of code reuse is a bit of a code smell, but let's see how this works out...